packages/taquito/src/operations/smart-rollup-originate-operation.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  OperationContentsAndResult,
+  OperationContentsAndResultSmartRollupOriginate,
+  OpKind,
+} from '@taquito/rpc';
+import { Context } from '../context';
+import { Operation } from './operations';
+import {
+  FeeConsumingOperation,
+  ForgedBytes,
+  GasConsumingOperation,
+  RPCSmartRollupOriginateOperation,
+  StorageConsumingOperation,
+} from './types';
+
+/**
+ *
+ * @description UpdateConsensusKeyOperation provides utility to fetch properties for Operation of kind UpdateConsensusKey
```

Comment:
small thing, update ConsensusKey to SmartRollupOriginate in the beginning and the end of the sentence